src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -73,6 +73,14 @@ public void AvoidMsdnReferencesValidation()
         }
 
         [Fact]
+        public void BooleanPropertiesValidation()
```

Comment:
I wasn't sure which positive test to add in this case, since many of the other tests already don't have boolean properties included. For example, "clean-complex-spec.json" does not have boolean properties and therefore doesn't return this warning. If you think that's not enough, just let me know and I can add something else.